### PR TITLE
making mlh logo, fonts responsive and making it mobile friendly

### PR DESCRIPTION
**Pull request by Apra487**
Hey @wrussell1999 I changed the font-size,made mlh logo responsive and fixed the allignment on the nav bar. 

Fixed the overlapping issue on mobile devices.

I am attaching the Before and After screenshot.

Before 
galaxy fold issue

After

after making the fix

### CODE REVIEW

**Comment by wrussell1999** [comment]
Looking good! Can you remove the comments from code @Apra487

**Comment by Apra487** [comment]
> Looking good! Can you remove the comments from code @Apra487

Hey @wrussell1999 removed all unnecessary comments as suggested by you.

File: _sass/navbar.scss [hunk 1]
```diff
@@ -16,8 +16,8 @@
 .nav-link {
     position: absolute;
     color: white;
-    top: 8px;
-    right: 8px;
+    top: 10px;
```
Comment (Korusuke) [inline _sass/navbar.scss:19]: very small nitpick but looks like you have an extra space here at the end

Comment (Apra487) [reply _sass/navbar.scss]: hey @Korusuke .Thank you for pointing it out. I removed it.

**Comment by Korusuke** [comment]
LGTM! Merging :rocket:

**Comment by Korusuke** [comment]
Is this linked with any issue @Apra487 ?

**Comment by Apra487** [comment]
> Is this linked with any issue @Apra487 ?

yes I guess . It is fixing https://github.com/MLH-Fellowship/portfolio-template/issues/11 this issue @Korusuke

**Comment by Korusuke** [comment]
Cools, seems like the issue will need more work so I will keep it open for now.
Just wanted to add one thing: If a PR address some issue we usually link it to the issue using a keyword like fixes, resolves, etc..

More information can be found here: https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**Comment by Apra487** [comment]
> Cools, seems like the issue will need more work so I will keep it open for now.
> Just wanted to add one thing: If a PR address some issue we usually link it to the issue using a keyword like fixes, resolves, etc..
> 
> More information can be found here: https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

Thank you @wrussell1999 and @Korusuke sir for your valuable feedback. I am just a beginner and this was my first pr in any major repository.

**Comment by wrussell1999** [comment]
Of course! :D @Apra487 

This is a nice warm up to get you familiar with PRs etc. You'll be an expert in a few weeks!

**Pull request merged**